src/components/publiq-ui/Alert.js

```diff
@@ -8,10 +9,22 @@ const AlertVariants = {
   WARNING: 'warning',
 };
 
+const StyledBootstrapAlert = styled(BootstrapAlert)`
+  &.alert-info {
+    background-color: #d9edf6;
+    border-color: #bce8ef;
+    color: #3e88ab;
+  }
```

Comment:
For simple components like `Alert` and `Button` we might be able to just roll our own with the same styles as the Bootstrap components? We're already explicitly setting these colours. All that's left to just roll our own would basically be some `padding`, `margin` and `border-radius`. (But do we even have a border-radius on alerts in the original design?) And then a transition and extra padding for dismissible alerts.

![Screenshot 2020-10-27 at 16 37 05](https://user-images.githubusercontent.com/959026/97325003-cbd4ee80-1872-11eb-92bb-0b9dbfcc00f2.png)

For pagination, modals etc I'd agree to keep re-using Bootstrap components because of their complexity. But in this case if we roll our own that might make it easier to make adjustments later?

Reply:
@bertramakers Your suggestion is to step away from Bootstrap as much as possible?

Reply:
I would just evaluate it per component. Like in this case, it's only a couple of extra CSS properties and then we don't need the Bootstrap component.

Not saying we have to ditch Bootstrap completely right now, but if we can recreate the same look with little effort maybe it's easier that way so if we ever want to move away from Bootstrap there's less refactoring.

Reply:
@geoffreydhuyvetters what do you think?

Reply:
The only reason why I wouldn't do this is to keep the built-in logic Bootstrap provides.

Reply:
For pagination, modal, etc definitely! For `Alert` and `Button`, I think that's relatively simple at first glance. For `Alert` it's the dismiss click handler that needs to hide/remove it, and for `Button` it's the click event that needs to call an injectable handler? Unless I'm oversimplifying it. Just basing this on what I see from reviewing the PRs. So if you both believe it would be much more work than what it's worth it's also okay to leave it as-is of course!

Reply:
Btw we should also overwrite the border-radius here, as we normally don't have one on alerts but Bootstrap adds one.

Can probably be done the same way as on `Button`.

<img width="738" alt="Screenshot 2020-10-28 at 10 05 59" src="https://user-images.githubusercontent.com/959026/97415277-4b130280-1905-11eb-9c03-09144c70bca6.png">